Add an example usage for ingest library to fetch statistics (#1349)
stellar/stellar-docs: docs/data/ingest-library/examples/README.mdx

---
title: Example Usages
sidebar_position: 10
---

This section showcases real-world use cases of the Ingest Library, focusing on key components like `ingest.LedgerTransaction`, `ingest.LedgerChange`, `ingest.LedgerTransactionReader`, and `ingest.ChangeReader`.

> These examples either taken in a Stellar Ledger (`xdr.LedgerCloseMeta`) or a ledger sequence range (start-end). Each of the examples listed here is complete, and can be taken as-is, compiled and run in your IDE/local dev.

Reviewer: ```suggestion
These examples take a Stellar Ledger (`xdr.LedgerCloseMeta`) or a ledger sequence range (start-end) as input. Each of the examples listed here is complete, and can be compiled and run as-is in your IDE/local dev.
```

Author: done

## Prerequisites

> Some of the examples listed here might require a local instance of captive-core. At a minimum, you will need a `stellar-core` binary that has been compiled for your platform. Refer "admin guide" for more details.

Author: "admin_guide" will point to the admin guide once that work is done. For now, this shud suffice

> Some of the examples listed here might require a local instance of captive-core. At a minimum, you will need a `stellar-core` binary that has been compiled for your platform. Refer "admin guide" for more details.

Reviewer 2: The captive-core terminology might be confusing for newcomers. I would rephrase as:

```suggestion
Some of the examples listed here might invoke the `stellar-core` binary. Refer to the "admin guide" for more details on how to compile `stellar-core` for your platform.
```

Author: done
